Layer 2/NCSBNMultiLicenseV1.0.json

```diff
@@ -0,0 +1,913 @@
+        },
+        "targetUrl": {
+          "type": "string",
+          "format": "uri",
+          "description": "URL of the target alignment. ",
+          "maxLength": 2048,
+          "x-stoplight": {
+            "id": "nhk4vxvoqbaxi"
+          }
+        },
+        "targetDescription": {
+          "type": "string",
+          "description": "Detailed description of the target alignment. ",
+          "maxLength": 2048,
+          "x-stoplight": {
+            "id": "fxj4tp7t2r0vj"
+          }
+        },
+        "targetCode": {
+          "type": "string",
+          "description": "Code of the target alignment. ",
+          "maxLength": 256,
+          "x-stoplight": {
+            "id": "h2w46yr7ceu2k"
+          }
+        },
+        "targetFramework": {
+          "type": "string",
+          "description": "The framework to which the target belongs. ",
+          "maxLength": 256,
+          "x-stoplight": {
+            "id": "pk3lr2rhf7ktj"
+          }
+        }
+      },
+      "required": [
+        "targetName",
+        "targetUrl"
+      ],
+      "x-stoplight": {
+        "id": "x0rwz35p9iq1z"
+      }
+    },
+    "PersonName": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "type": {
+          "type": "string",
+          "default": "PersonName",
+          "x-stoplight": {
+            "id": "u10ulnqfoytqe"
+          }
+        },
+        "givenName": {
+          "type": "string",
+          "description": "The given name or first name of the person.",
+          "maxLength": 64,
+          "x-stoplight": {
+            "id": "uq2ghdlwbo9ur"
+          }
+        },
+        "familyName": {
+          "type": "string",
+          "description": "The family name or surname of the person.",
+          "maxLength": 64,
+          "x-stoplight": {
+            "id": "es3hve994cso3"
+          }
+        },
+        "middleName": {
+          "type": "string",
+          "description": "The middle name of the person.",
+          "maxLength": 256,
+          "x-stoplight": {
+            "id": "6ddvig03dbtfp"
+          }
+        },
+        "namePrefix": {
+          "type": "string",
+          "description": "A prefix to the name such as Dr or Sir.",
+          "maxLength": 8,
+          "x-stoplight": {
+            "id": "qi3qtgknjcg6z"
+          }
+        },
+        "nameSuffix": {
+          "type": "string",
+          "description": "A suffix to the name such as Jr. or Sr.",
+          "maxLength": 16,
+          "x-stoplight": {
+            "id": "gs9oieevkljlg"
+          }
+        }
+      },
+      "required": [
+        "givenName",
+        "familyName"
+      ],
+      "x-stoplight": {
+        "id": "dehjkrrkhiwd6"
+      }
+    },
+    "AdvancedPractice": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "properties": {
+        "FocusSpecialty": {
+          "type": [
+            "string",
+            "null"
```

Comment:
we are deliberately allowing a null value, in addition to the property not being there? There are many examples of this. That will fulfill the requirement property. Meaning we are saying that StateAbbreviation is required, but a valid value for it is null? That is deliberate?

Reply:
Yes. We are following NCSBN's schema conventions here.